src/core/BufferAttribute.js

```diff
@@ -42,6 +43,13 @@ class BufferAttribute {
 
 	}
 
+	get updateRange() {
+
+		console.warn( 'BufferAttribute: "updateRange" is deprecated and removed in r169. Use "addUpdateRange" instead.' );
```

Comment:
Do you mind writing it like so:
```js
console.warn( 'THREE.BufferAttribute: "updateRange" is deprecated and removed in r169. Use ".addUpdateRange()" instead.' );
```
This adds the `THREE` namespace and makes it more clear that `addUpdateRange()` is a method. `InterleavedBuffer` needs the same change.

Reply:
Done!